server/src-lib/Hasura/RQL/DDL/Remote/Validate.hs

```diff
@@ -141,38 +144,50 @@ validateRemoteArguments expectedArguments providedArguments permittedVariables =
 
 -- | Validate a value against a type.
```

Comment:
All these changes for `validateType` look good to me. I didn't think it would be complicated. :+1:

Reply:
A quick `hindent` over this function would go nicely, but not necessary, just handy.

Reply:
Yeah, `hindent` is awesome. I always thought stylish-haskell would do the needful reformatting.

Reply:
stylish-haskell just reformats your imports and extensions and some other not-so-useful stuff, or harmful stuff like doing alignment. I'm not sure why anybody even uses it to be honest. :man_shrugging: